cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -54,6 +54,15 @@ spec:
         # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
         # - name: AGENT_TOLERATION_KEY
         #   value: "<KeyOfTheTaintToTolerate>"
+        # (Optional) Rook Agent NodeAffinity Label Key. 
+        # Set this to the key of the Node Label to match.
+        # - name: AGENT_NODE_LABEL_KEY
+        #   value: "<KeyOfTheLabelToMatch>"
+        # (Needed if AGENT_NODE_LABEL_KEY is set ) Rook Agent NodeAffinity Label Value.
+        # Set this to the value of the Node Label to match.(Comma separated values if multiple values 
+        # needs to be matched against single key)
+        # - name: AGENT_NODE_LABEL_VALUE
```

Comment:
If we keep them separate, since multiple values are possible, should we name this `AGENT_NODE_LABEL_VALUES`? (and also the other discovery and helm chart names could be changed to plural for the same reason)

Reply:
ack.